pkg/entpgx/pgxpool_dialect_driver.go

```diff
@@ -0,0 +1,260 @@
+package entpgx
+
+import (
+	"context"
+	stdsql "database/sql"
+	"entgo.io/ent/dialect"
+	"entgo.io/ent/dialect/sql"
+	"fmt"
+	"github.com/jackc/pgx/v5"
+	"github.com/kong/pg-aurora-client/pkg/pool"
+)
+
+func NewPgxPoolDriver(pool pool.PGXConnPool) dialect.Driver {
+	return &EntPgxpoolDriver{
+		pool: pool,
+	}
+}
+
+type EntPgxpoolDriver struct {
+	pool pool.PGXConnPool
+}
+
+func (e *EntPgxpoolDriver) Exec(ctx context.Context, query string, args, result any) error {
+	var _ stdsql.Result
```

Comment:
question: What is this here for?

Reply:
Ent requires us to implement dialect.ExecQuerier interface for both Pool level (without exposing the transaction) as well as with Tx level. Since we want to be interface compatible for situations where client is not expected to manage the transaction, that is the reason for these two implementations.

Reply:
I was referring to the unnamed variable.

Reply:
Got it. Yeah, for some reason, I had put it there and then forgot to remove it. Done